src/mds/MDSRank.cc

```diff
@@ -715,6 +715,7 @@ void MDSRankDispatcher::tick()
   sessionmap.update_average_session_age();
 
   if (is_active() || is_stopping()) {
+    server->recall_client_state(nullptr, Server::RecallFlags::ENFORCE_MAX);
     mdcache->trim();
     mdcache->trim_client_leases();
     mdcache->check_memory_usage();
```

Comment:
check_memory_usage also call recall_client_state().  we recall_client_state() (with different mode) twice here. Is this correct?